library/clojure

```diff
@@ -1,36 +1,86 @@
 Maintainers: Paul Lam <paul@quantisan.com> (@Quantisan),
-             Wes Morgan <wesmorgan@icloud.com> (@cap10morgan),
-             Kirill Chernyshov <delaguardo@gmail.com> (@DeLaGuardo)
+             Wes Morgan <wesmorgan@icloud.com> (@cap10morgan)
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
```

Comment:
Unfortunately the `openjdk` images you are `FROM` now have a very limited set of architectures 😞 (the cause of the travis build failure).

A few, like `openjdk:11-slim-buster` have `arm64v8`, but most are just `amd64` or Windows `amd64`.
https://github.com/docker-library/official-images/blob/beb43151845ae460f10ffbc1d8fd8a3b40760015/library/openjdk#L94-L95

Reply:
Note: the Travis output shows which architectures you have here that are not supported by that parent image.

Reply:
OK I’ll cut back on the architectures we’re trying to build. Thanks @yosifkit!